### PR TITLE
[zh] Sync 1.18.7 release note and 1.18 eol into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync 1.18.7 release note and 1.18 eol into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+该版本包含的错误修复用于提高稳健性。
+
+本发布说明描述了 Istio 1.18.6 和 Istio 1.18.7 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了一个当选择通配符服务时（例如在 `ServiceEntry` 中），
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18.7/index.md:18]:
```suggestion
- **修复** 修复了（例如在 `ServiceEntry` 中）通过通配符选中多个服务时，
```

File: content/zh/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ([Issue #45415](https://github.com/istio/istio/issues/45415))
+
+- **修复** 修复了 `istioctl proxy-config ecds` 未显示所有 `EcdsConfigDump` 的问题。
+
+- **修复** 修复了新端点可能无法被发送到代理的问题。
+  ([Issue #48373](https://github.com/istio/istio/issues/48373))
+
+- **修复** 修复了使用 Stackdriver 安装并使用自定义配置会阻止 Stackdriver 被启用的问题。
+
+- **修复** 修复了长连接、TCP 字节和 gRPC 可能导致代理内存泄漏的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18.7/index.md:29]:
```suggestion
- **修复** 修复了生命期长的连接、TCP 字节和 gRPC 可能导致代理内存泄漏的问题。
```

File: content/zh/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+该版本包含的错误修复用于提高稳健性。
+
+本发布说明描述了 Istio 1.18.6 和 Istio 1.18.7 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了一个当选择通配符服务时（例如在 `ServiceEntry` 中），
+  `VirtualService` 中重叠的通配符主机会产生不正确的路由配置的错误。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18.7/index.md:19]:
```suggestion
  `VirtualService` 中因通配符造成的重叠主机会产生不正确路由配置的错误。
```
not sure the translation is correct or not, cc @hanxiaop

**Review by windsonsea — COMMENTED** [review]
直译可能有点问题，这 2 个词好像不常见：

- 通配符服务
- 通配符主机

File: content/zh/news/releases/1.18.x/announcing-1.18.7/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  ([Issue #45415](https://github.com/istio/istio/issues/45415))
+
+- **修复** 修复了 `istioctl proxy-config ecds` 未显示所有 `EcdsConfigDump` 的问题。
+
+- **修复** 修复了新端点可能无法被发送到代理的问题。
+  ([Issue #48373](https://github.com/istio/istio/issues/48373))
+
+- **修复** 修复了使用 Stackdriver 安装并使用自定义配置会阻止 Stackdriver 被启用的问题。
+
+- **修复** 修复了生命期长的连接、TCP 字节和 gRPC 可能导致代理内存泄漏的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.18.x/announcing-1.18.7/index.md:29]:
```suggestion
- **修复** 修复了 TCP 和 gRPC 长连接导致代理内存泄漏的问题。
```

**Comment by hanxiaop** [comment]
FYI 1.18.7 is out https://istio.io/latest/news/releases/1.18.x/announcing-1.18.7/, it's safe to merge the cherry-pick one as well.

**Review by windsonsea — APPROVED** [review]
lgtm, thanks.

**Pull request merged**